hazelcast-client-new/src/main/java/com/hazelcast/client/config/XmlClientConfigBuilder.java

```diff
@@ -264,11 +256,36 @@ private void handleNearCache(Node node) {
             } else if ("local-update-policy".equals(nodeName)) {
                 NearCacheConfig.LocalUpdatePolicy policy = NearCacheConfig.LocalUpdatePolicy.valueOf(value);
                 nearCacheConfig.setLocalUpdatePolicy(policy);
+            } else if ("eviction".equals(nodeName)) {
+                nearCacheConfig.setEvictionConfig(getEvictionConfig(child));
             }
         }
         clientConfig.addNearCacheConfig(name, nearCacheConfig);
     }
 
+    private EvictionConfig getEvictionConfig(final org.w3c.dom.Node node) {
```

Comment:
can you remove `org.w3c.dom` and import `org.w3c.dom` to header of class. only `Node` more readable.